frontend/views/containers/proposals/AddMembers.vue

```diff
@@ -17,12 +17,10 @@
             type='text'
             name='mincome'
             required=''
-            :class="{ 'error': $v.form.member.$error }"
-            v-model='form.member'
-            @keyup.enter='next'
```

Comment:
Just double-checking, since this removal of 2 lines is a change that goes beyond merely fixing the quotes, that you meant to do this? You mean to remove both the `v-model` binding and the key binding?

Reply:
It was the scouting rule, but it really doesn't matter because AddMembers is not functional at all ATM. i'll put it back